pkgs/development/python-modules/jax-cuda12-plugin/default.nix

```diff
@@ -39,42 +39,42 @@ let
     "3.10-x86_64-linux" = getSrcFromPypi {
       platform = "manylinux2014_x86_64";
       dist = "cp310";
-      hash = "sha256-nULpmc1k3VZ8FJ7Wj3k5K6iGRDZCGLtjbNzvoBl8kv4=";
+      hash = "sha256-D0Q6azcpjt+weW/NvR+GzoWksIS2vT8fUKT7/Wfe2Gs=";
     };
     "3.10-aarch64-linux" = getSrcFromPypi {
       platform = "manylinux2014_aarch64";
       dist = "cp310";
-      hash = "sha256-AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=";
+      hash = "";
```

Comment:
I didn't understand this in the original PR, why are the aarch64 hashes empty?

Reply:
But the fact that it's in `srcs`, just with an empty hash breaks the throw logic, right?

https://github.com/NixOS/nixpkgs/blob/efe536798ded4bb3bb9ea94847514c37b731d695/pkgs/development/python-modules/jax-cuda12-plugin/default.nix#L86-L89

Do we believe upstream will eventually upload aarch64 binaries and this a placeholder until then?